modules/littlefs/zephyr_lfs_config.h

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) 2017, Arm Limited. All rights reserved.
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Origin of this file is the LittleFS lfs_util.h file adjusted for Zephyr
+ * specific needs.
```

Comment:
It's not possible to #include lfs_util.h instead of copying it is it ?

Reply:
Litttlefs Project stands in its lfs_util.h: 
> // Users can override lfs_util.h with their own configuration by defining
// LFS_CONFIG as a header file to include (-DLFS_CONFIG=lfs_config.h).

so this is OK